core/src/main/python/wlsdeploy/tool/util/wlst_helper.py

```diff
@@ -124,6 +126,11 @@ def is_set(self, attribute):
                                                     self.__get_exception_mode(e), _format_exception(e), error=e)
             self.__logger.throwing(class_name=self.__class_name, method_name=_method_name, error=pwe)
             raise pwe
+        except Throwable, e:
+            # isSet() throws a Java error for /ResourceManagement attribute in 14.1.1, return False to halt discovery
```

Comment:
done:
```
            # isSet() throws a Java error for /ResourceManagement attribute in 14.1.1.
            # in this case, return False to prevent further processing of this attribute.
```